base/docs/basedocs.jl

```diff
@@ -692,7 +692,7 @@ See the manual section on [control flow](@ref man-conditional-evaluation) for mo
 ```
 julia> x = 1; y = 2;
 
-julia> println(x > y ? "x is larger" : "y is larger")
+julia> x > y ? println("x is larger") : println("y is larger")
```

Comment:
why this change? I thought the old version was more canonical

Reply:
In the old one, there is nothing to run in the two expressions. `ifelse` would return the same answer.